README.md

```diff
@@ -184,12 +184,11 @@ USER root
 
 RUN apk add --update --virtual .build-deps \
         sudo build-base ruby-dev \
-
- # cutomize following instruction as you wish
- && sudo -u fluent gem install \ 
+ `# customize following instruction as you wish` \
```

Comment:
Shell scripts don't support inline comments, so I'm using the '`' (called backticks) to perform [command substitution that evaluates to a no-op](http://stackoverflow.com/questions/2524367/inline-comments-for-bash). Text between backticks is executed and replaced by the output of the command. Since the command is just a line terminating comment, the output of the command is an empty string. I agree it's obscure and not ideal.

Better solutions would probably be to either move all the comments to precede the RUN, or just RUN a helper script where line terminating comments could be intermixed.